### ISSUE TITLE
Normalization layer

### ISSUE THREAD

**Issue by CanPhi** [albermax/innvestigate#90]
Hi, 

I am wondering which of the methods proposed work with networks that use normalization layers or whether there are any other constraints to the model architecture. From the PatternNet paper it appears like normalzation (either batch or instance normalization layers) cannot be used for networks that are to be analyzed with PatternNet. Is this the case?

If so, which methods are suited to be applied to neural networks with normalization layers?

Thanks!
Philippe

**Comment by enryH** [comment]
If deep taylor is not yet supported, how is the LRPAlpha1Beta0 supported with Batch Normalization?
